include/dfsinternal.h

```diff
@@ -52,6 +55,30 @@ typedef struct dfs_open_file_s
     uint32_t cart_start_loc;
 } dfs_open_file_t;
 
+/** @brief DFS File Entry */
+typedef struct dfs_file_entry_s {
+    /** @brief Hash of file path */
+    uint32_t path_hash;
+    /** @brief Top 12 bits: length; lowest 20 bits: offset */
+    uint32_t path_ofs;
+    /** @brief Data offset for file */
+    uint32_t data_ofs;
+    /** @brief Data length for file */
+    uint32_t data_len;
+} dfs_file_entry_t;
+
+/** @brief Data for DFS file lookup */
```

Comment:
In general, please try to elaborate a bit more in comments. For instance, here explains what is this "lookup" data structure, what it contains and why it's used.

Reply:
Have given purpose of dfs_lookup_t.